stable/redis/templates/redis-master-statefulset.yaml

```diff
@@ -343,6 +343,9 @@ spec:
       - name: redis-password
         secret:
           secretName: {{ template "redis.secretName" . }}
+          items:
+          - key: {{ template "redis.secretPasswordKey" . }}
+            path: redis-password
```

Comment:
Did you test this? I found the error below in the master pods:
```
cat: /opt/bitnami/redis/secrets/redis-password: No such file or directory
```
I used this command:
```
$ kubectl create secret generic existing-redis-secret --from-literal=my-redis-key-for-password=my-custom-password
$ helm install stable/redis --name redis \
    --set existingSecret=existing-redis-secret \
    --set existingSecretPasswordKey=my-redis-key-for-password \
    --set usePasswordFile=true
```

Reply:
Yeah, it's been working in my testing. I ran it like this without problems:

```
kubectl create secret generic existing-redis-secret --from-literal=my-redis-key-for-password=my-custom-password
helm upgrade --install redis . --set existingSecret=existing-redis-secret --set existingSecretPasswordKey=my-redis-key-for-password --set usePasswordFile=true
```

Reply:
I don't know what happened but I retried and it worked this time.. Sorry for the noise!